src/pages/executive-board.js

```diff
@@ -0,0 +1,227 @@
+    width: '100%',
+    height: '450px',
+    transition: 'all 0.3s ease-out',
+    [theme.breakpoints.down('xs')]: {
+      height: '300px',
+    },
+    background: '#000',
+    borderRadius: '12px',
+  },
+  bottomText: {
+    position: 'absolute',
+    bottom: 0,
+    width: '100%',
+    color: '#FFF',
+    height: '100px',
+    transition: 'all 0.3s ease-out',
+    borderRadius: '12px',
+    background: 'rgba(0, 0, 0, 0.7)',
+    padding: theme.spacing(0.5),
+  },
+  socialIconsContainer: {
+    position: 'absolute',
+    top: '42%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    visibility: 'hidden',
+  },
+  socialIcon: {
+    background: 'transparent',
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+  icon: {
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+}))
+
+function Something(props) {
+  const classes = useStyles(props)
+  const theme = useTheme()
+
+  const committees = ['UNSC', 'UNODC', 'DISEC', 'UNCSW', 'WHO', 'AIPPM']
+  const { image, bgImage } = useStaticQuery(graphql`
+    query {
+      image: file(relativePath: { eq: "banners/about.jpg" }) {
+        sharp: childImageSharp {
+          fluid(maxWidth: 1080) {
+            ...GatsbyImageSharpFluid_withWebp
+          }
+        }
+      }
+      bgImage: file(relativePath: { eq: "pages-background.png" }) {
+        sharp: childImageSharp {
+          fluid(maxWidth: 1080) {
+            ...GatsbyImageSharpFluid_withWebp
+          }
+        }
+      }
+    }
+  `)
+
+  return (
+    <Wrapper>
+      <Helmet>
+        <link
+          href='https://fonts.googleapis.com/css?family=Rubik&display=swap'
+          rel='stylesheet'
+        />
+      </Helmet>
+      <Banner
+        backgrounds={[
+          `linear-gradient(${theme.palette.glare.main}, ${theme.palette.glare.main})`,
+          image.sharp.fluid,
+        ]}
+        height='50vh'
+      >
+        <Typography
+          color='primary'
+          component='h2'
+          className={classes.munTextProperty}
+        >
+          EXECUTIVE BOARD
+        </Typography>
+        <Typography className='text-white' variant='h5'>
+          JECRC MUN 2020
+        </Typography>
+      </Banner>
+      <BackgroundImage
+        className='flex flex-col justify-center items-center'
+        fluid={bgImage.sharp.fluid}
+        durationFadeIn={50}
+      >
+        <Grid container justify='center' className={classes.root}>
+          {committees.map((name, index) => (
+            <Grid key={index} container className='w-full' justify='center'>
+              <Grid item xs={12} className='text-center py-10'>
+                <Typography style={{ color: '#000a2a' }} variant='h4'>
+                  {name}
+                </Typography>
+                <img
+                  src='/images/line.png'
+                  className='mx-auto'
+                  alt='---------------------'
+                />
+              </Grid>
+              {[1, 2].map(value => (
```

Comment:
create a sample recorder for EB in the data file and use that, 
this is very irrelevant and unprofessional way of doing work